docker/dockerbuild.sh

```diff
@@ -12,14 +11,3 @@ bower --allow-root install
 # Compress images and build
 yarn run build optimize-images
 yarn run build
-cp -r ./build/. /srv/www
-
-# Uninstall installed build dependencies and remove cache
```

Comment:
Because the `builder` container is thrown away, only the artifacts that it built are used in the final image.